Sources/StreamChatSwiftUI/ChatChannel/MessageList/MessageContainerView.swift

```diff
@@ -7,6 +7,8 @@ import StreamChat
 import SwiftUI
 
 public struct MessageContainerView<Factory: ViewFactory>: View {
+    @EnvironmentObject var channelViewModel: ChatChannelViewModel
+    @EnvironmentObject var messageViewModel: MessageViewModel
```

Comment:
I have a stashed commit that uses `@EnvinronmentValues` instead, like we do with the `@Environment(\.channelTranslationLanguage)`. Ofc, there are pros and cons here:

**@EnvironmentObject:**
+ No Optionals
- Unsafe, can cause crashes easily if a customer is using views directly and do not provide the env object

**@Environment:**
+ Safer
- More boilerplate code, to fallback to previous logic when the view model is optional